.github/workflows/posix.yml

```diff
@@ -277,53 +278,35 @@ jobs:
         ./script/test generate_coverage gcc
     - uses: actions/upload-artifact@v2
       with:
-        name: cov-posix-pty
+        name: cov-pty-linux-${{ matrix.DAEMON }}
         path: tmp/coverage.info
 
-  posix-pty-daemon:
-    runs-on: ubuntu-20.04
+  pty-macos:
+    runs-on: macos-10.15
+    strategy:
+      matrix:
+        DAEMON: [0, 1]
     env:
-      COVERAGE: 1
-      DAEMON: 1
+      DAEMON: ${{ matrix.DAEMON }}
     steps:
     - uses: actions/checkout@v2
-      with:
-        submodules: true
     - name: Bootstrap
       run: |
-        sudo rm /etc/apt/sources.list.d/* && sudo apt-get update
-        sudo apt-get --no-install-recommends install -y socat expect lcov net-tools
-        cd /tmp
-        wget https://github.com/obgm/libcoap/archive/bsd-licensed.tar.gz
-        tar xvf bsd-licensed.tar.gz
-        cd libcoap-bsd-licensed
-        ./autogen.sh
-        ./configure --prefix= --exec-prefix=/usr --with-boost=internal --disable-tests --disable-documentation
-        make -j2
-        sudo make install
+        brew install ninja
     - name: Build
       run: |
-        ./bootstrap
         script/check-posix-pty build
     - name: Run
       run: |
         script/check-posix-pty check
-    - name: Generate Coverage
-      run: |
-        ./script/test generate_coverage gcc
-    - uses: actions/upload-artifact@v2
-      with:
-        name: cov-posix-pty-daemon
-        path: tmp/coverage.info
 
   upload-coverage:
     needs:
     - expects-linux
     - posix-cli
     - posix-ncp
     - posix-ncp-rcp-migrate
-    - posix-pty
-    - posix-pty-daemon
+    - pty-linux
```

Comment:
```suggestion
    - pty-linux
    - pty-macos
```

Reply:
I failed to get code coverage on macOS yet.

Reply:
Then it would be no difference whether or not `pty-macos` is added here. I am fine to both way.